pkgs/development/python-modules/simpleldap/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage, ldap, pep8, pytest, tox, lib, fetchFromGitHub, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "simpleldap";
+  version = "0.8";
+
+  src = fetchFromGitHub {
```

Comment:
What's the reason you switched to `fetchFromGitHub`? Please add an inline comment.